src/Spryker/Glue/AppKernel/AppKernelDependencyProvider.php

```diff
@@ -9,6 +9,7 @@
 
 use Spryker\Glue\AppKernel\Dependency\Facade\AppKernelToAppKernelFacadeBridge;
 use Spryker\Glue\AppKernel\Plugin\RequestValidator\BodyStructureValidatorPlugin;
+use Spryker\Glue\AppKernel\Plugin\RequestValidator\ConfigurationValidatorPlugin;
 use Spryker\Glue\AppKernel\Plugin\RequestValidator\HeaderValidatorPlugin;
```

Comment:
The plugins are from AppKernel for AppKernel, usually, plugins are external classes provided by a project or another module. In this case plugins can be replaced by AppKernel classes enabled by default or configured via config.